x-pack/plugins/security/server/session_management/session_index.ts

```diff
@@ -37,31 +37,33 @@ const SESSION_INDEX_TEMPLATE_VERSION = 1;
 /**
  * Returns index template that is used for the current version of the session index.
  */
-export function getSessionIndexTemplate(indexName: string) {
+export function getSessionIndexTemplate(templateName: string, indexName: string) {
   return Object.freeze({
+    name: templateName,
     index_patterns: [indexName],
-    order: 1000,
```

Comment:
__note:__ there is no top level `order` property in the composable index templates anymore. There is a `priority` instead, but I cannot think of a good reason why we'd want to set it explicitly. Thoughts?

Reply:
Reading the [docs](https://www.elastic.co/guide/en/elasticsearch/reference/7.16/indices-put-template.html#multiple-templates), I agree I don't think we need this. We shouldn't expect / support other index templates that overlap with this one.